src/System.Runtime.Extensions/src/System/Net/WebUtility.cs

```diff
@@ -666,6 +676,13 @@ internal String GetString()
                 else
                     return String.Empty;
             }
+
+            internal void Dispose()
+            {
+                ArrayPool<char>.Shared.Return(_charBuffer);
+                if (_byteBuffer != null)
+                    ArrayPool<byte>.Shared.Return(_byteBuffer);
```

Comment:
As a struct this won't provide perfect protection, but it'd be good to null out both of these fields, regardless.  Double-dispose of this struct would end up returning these buffers multiple times.